app/assets/javascripts/productions.js

```diff
@@ -0,0 +1,82 @@
+$(document).ready(function(){
+	console.log("document ready")
+	toggleMenus();
+    bindListeners();
+});
+
+//------------------------------------------------
+// Bootstrap Functions
+//------------------------------------------------
+var toggleMenus = function(){
+	$(".inventory-toggle").click(function(e){
+    	e.preventDefault();
+      $(".inventory-tab").slideToggle();
+    });
+
+ //    $(".print-toggle").click(function(e){
+	//   e.preventDefault();
+ //    $(".print-tab").slideToggle();
+ //    });
+
+    $(".library-toggle").click(function(e){
+    	e.preventDefault();
+      $(".library-tab").slideToggle();
+    });
+};
+
+//-------------------------------------------------
+// All listeners bound together
+//-------------------------------------------------
+var bindListeners = function(){
+  deleteButtonListener();
+  editButtonListener();
+};
+
+//-------------------------------------------------
+// Listeners
+//-------------------------------------------------
+var deleteButtonListener = function(){
+  $('#channel-hookup').on('click', '.delete-button', function(e){
+    e.preventDefault();
+    console.log("delete row: " + this);
+    deleteRow(this);
+  });
+};
+
+var editButtonListener = function(){
+  $('#channel-hookup').on('click', '.edit-button', function(e){
+    e.preventDefault();
+    console.log("edit row: " + this);
+    editRow(this);
+  });
+};
+//-------------------------------------------------
+// Functions
+//-------------------------------------------------
+var deleteRow = function(path){
+  $.ajax({
+    method: "DELETE",
+    url: path,
+    dataType: 'json'
+  }).done(function(response){
+    console.log("Removing item:" );
+    $('#CH-Row' + response.rowNumber).remove();
+    $('#CK-Row' + response.rowNumber).remove();
+    $('#DM-Row' + response.rowNumber).remove();
+    $('#IS-Row' + response.rowNumber).remove();
+  }).fail(console.log("ajax delete call failed "))
+};
+
+var editRow = function(path){
+  $.ajax({
+    method: "GET",
+    url: path,
+    dataType: 'json'
+  }).done(function(response){
+    console.log("Removing item:" );
+    $('#CH-Row' + response.rowNumber).remove();
+  }).fail(console.log("ajax delete call failed "))
+};
```

Comment:
looks redundant with `deleteRow`. perhaps `deleteRow` should take an argument that specifies if the 4 elements should be deleted or just one as in `editRow`